bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/InitializeToolsStartup.java

```diff
@@ -56,10 +58,8 @@ public class InitializeToolsStartup implements IStartup
 	private static final String IDF_INSTALLED_LIST_KEY = "idfInstalled"; //$NON-NLS-1$
 	private static final String PYTHON_PATH = "python"; //$NON-NLS-1$
 	private static final String IDF_PATH = "path"; //$NON-NLS-1$
-	private static final String IS_INSTALLER_CONFIG_SET = "isInstallerConfigSet"; //$NON-NLS-1$
 	private static final String DOC_URL = "\"https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/partition-tables.html?highlight=partitions%20csv#creating-custom-tables\""; //$NON-NLS-1$
-
-	private String newIdfPath;
+	private static final String LAST_MODIFIED_ESP_IDF_JSON_FILE = "lastModifed-esp_idf.json"; //$NON-NLS-1$
```

Comment:
Name is bit misleading, my initial impression was you're trying to store lastmodified timestamp in the `lastModifed-esp_idf.json` file. How about something like this  `esp_idf_json_lastmodified_timestamp` or similar?